CONTRIBUTING.md

```diff
@@ -38,22 +38,40 @@ Tests are stored in [tests](/tests) directory:
     for remote communication => offline in the CI
   - prefer [requre](https://github.com/packit/requre) instead of mocking
 
+Running tests:
+
 Running tests locally:
 
-    make check_in_container
+Make sure you have the following packages:
+
+    python3-pytest
+    python3-requre
+    python3-flexmock
+
+installed, then:
+
+    make check
+
+Running tests in a container:
+
+Make sure you have the podman package installed, then:
+
+    make build-test-image
+    make check-in-container
```

Comment:
@bcrocker15 can you make these lines more similar to [these](https://github.com/packit/packit-service-fedmsg/pull/43/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R11-R29) please? One reason for this ask, is that additionally to install these 3 packages, package (that is `packit` dependencies) should also be installed before `make check` would work, so the current instructions are incomplete.